WeScan/Edit/EditImageViewController.swift

```diff
@@ -0,0 +1,203 @@
+//
+//  EditImageViewController.swift
+//  WeScan
+//
+//  Created by Chawatvish Worrapoj on 7/1/2020
+//  Copyright © 2020 WeTransfer. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+/// A method that your delegate object must implement to get cropped image.
```

Comment:
This is the documentation for the function, but it is on the protocol.
We need to have documentation for both.

Reply:
Hi @Boris-Em 
Apologies for the late reply.

This PR I would like to split the camera and edit image view out of other components.
I think we should have an individual view of a capture object and edit images after capture.
About other components like flash button, auto/manual detect, or shutter button people can customize it.

Reply:
> This is the documentation for the function, but it is on the protocol.
> We need to have documentation for both.

Means I have to document on code right?

Reply:
Yes, on the protocol.

Reply:
I changed it las 2 days. Thank you 🙏🏻